App/src/main/webapp/style.css

```diff
@@ -71,4 +71,121 @@ footer {
   color: var(--text-secondary);
   font-weight: 600;
   margin-bottom: 40px;
-}
\ No newline at end of file
+}
+
+        #slider{
+            overflow: hidden;
+            z-index: 1;
+            position: relative;
+            height: 400px;
+        }
+
+        #slider figure{
+            position: relative;
+            width: 500%;
+            margin: 0;
+            left: 0;
+            animation: 20s slider infinite;
+        }
+        #slider figure img {
+            max-height: 400px;
+            min-height: 400px;
+            object-fit: cover;
+            width: 20%;
+            float: left;
+        }
+        #title{
+            z-index: 10;
+            position: relative;
+            transform: translateY(-200px);
+        }
+
+@keyframes slider {
+	0% {
+		left: 0;
+	}
+	20% {
+		left: 0;
+	}
+	25% {
+		left: -100%;
+	}
+	45% {
+		left: -100%;
+	}
+	50% {
+		left: -200%;
+	}
+	70% {
+		left: -200%;
+	}
+	75% {
+		left: -300%;
+	}
+	95% {
+		left: -300%;
+	}
+	100% {
+		left: -400%;
+	}  
+}
+#btn{
+    padding: 10px 30px;
+    font-size: 1.2rem;
+    color: white;
+    background-color: #a13237;
+    border: none;
+    border-radius: 10px;
+    z-index: 10;
+    position: relative;
+    transform: translateY(-235px);
+    left: 500px;
+    /*right:500px;*/
+    margin: center;
+    display:flex;
+    width:175px
+}
+    .wrapper{
+        display:flex;
+        flex-direction: column;
+        align-items: center;        
+        justify-content: center;
+    }
+    #title {
+        z-index: 10;
+        position: relative;
+        transform: translateY(-250px);
+        left: 280px;
+        padding: 20px 50px;
+        font-size: 50px;
+        color: white;
+        -webkit-text-stroke-width: 2px;
+        -webkit-text-stroke-color: #2b2727;
+        font-weight: bold;
+    } 
+
+    #info-title{
+        font-size: 40px;
+        color: var(--text-secondary);
+        font-weight: 600;
+        margin-bottom: 40px;
+        margin-left: 0px;
+    }
+
+    #info-p{
+        font-size: 20px;
+        color: var(--text-secondary);
+        font-weight: 600;
+        margin-bottom: 30px;
+        margin-right: 80px;
+        margin-left: 80px;
+        /*text-indent: 10%;*/
+    }
+
+    #info-p2{
+        font-size: 20px;
+        color: var(--text-secondary);
+        font-weight: 600;
+        margin-bottom: 15px;
+        margin: center;
+    }
```

Comment:
Indentation is a bit weird here, as well as with the ids above the keyframes. Nothing major though